luigi/scheduler.py

```diff
@@ -224,7 +224,7 @@ class Worker(object):
     Structure for tracking worker activity and keeping their references.
     """
 
-    def __init__(self, worker_id, last_active=None):
+    def __init__(self, worker_id, last_active=time.time()):
```

Comment:
oh yea it is at module initialization time! Not worker creation time. But wait, if we just want to fix the type error, it might as well be `last_active=0.0`. But then `prune()` would kill the worker prematurely?

@weidongtoml, your thoughts?